app/src/main/java/ch/blinkenlights/android/vanilla/DraggableRow.java

```diff
@@ -61,6 +62,7 @@ public DraggableRow(Context context, AttributeSet attrs) {
 	public void onFinishInflate() {
 		mCheckBox  = (CheckedTextView)this.findViewById(R.id.checkbox);
 		mTextView  = (TextView)this.findViewById(R.id.text);
+		mDurationView = findViewById(R.id.duration);
```

Comment:
I think that this view should be set to 'gone' by default:

Please change `draggable_row.xml` to set it to gone (see comment below) and add
```
mDurationView.setVisibility(View.VISIBLE);
```

to `case LAYOUT_DRAGGABLE:` (~line 86)

Reply:
Done, but PlaylistAdapter.java also uses `DraggableRow.LAYOUT_DRAGGABLE`, so I decided to show the song duration for playlists too